config.lua

```diff
@@ -334,6 +334,7 @@ Config.CarPounds = {
 		Spawner =  vector3(994.5, -2289.2, 29.5),
 		Heading = 174.55,
 		Hidden = true
+	}
```

Comment:
There are several errors in config.lua that needs to be addressed.

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L64

Extra bracket

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L242

Missing comma.

Please double check everything.